### ISSUE TITLE
Enable debugging when mocking lambda with graphql api

### ISSUE THREAD

**Issue by tnguyen01** [aws-amplify/amplify-cli#2063]
I can debug/hit break points in lambda functions being invoked/mocked directly but not when running graphql api & lambda locally using `amplify mock`. Looks like it's because the child process to invoke lambdas is forked with empty `execArgv` - https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-util-mock/src/utils/lambda/invoke.ts#L9 

Was this done intentionally to avoid some issue? Can we add `--inspect` arg there to allow debugging?

**Comment by Bulletninja** [comment]
seems related to https://github.com/aws-amplify/amplify-cli/issues/2482

**Comment by cyrfer** [comment]
Yes!! Please make it easy to debug Amplify API Lambdas in VS Code with `amplify mock`!

`amplify mock function ABC` is not very helpful. I might as well run a mocha test if I need to specify an event object interactively in the terminal.
